adapter/prometheus/BUILD

```diff
@@ -31,6 +31,7 @@ go_test(
         "//adapter/prometheus/config:go_default_library",
         "//pkg/adapter:go_default_library",
         "//pkg/adapter/test:go_default_library",
+        "//template/metric:go_default_library",
```

Comment:
no sure how this compiled before with a missing dependency?
Was it being transitively included ?

Reply:
That's likely, so it would have caused a build error down the road if the intermediate target is removed.